scipy/stats/_continuous_distns.py

```diff
@@ -5654,6 +5655,455 @@ def _stats(self, a, b, moments='mv'):
+            -0.00026400886380,
+            -0.00008609203582,
+            +0.00002938828486,
+            -0.00000628851988,
+            +0.00000105653097,
+            -0.00000013289345,
+            +0.00000000661774,
+            +0.00000000287306,
+            -0.00000000132848,
+            +0.00000000037590,
+            -0.00000000008510,
+            +0.00000000001599,
+            -0.00000000000234,
+            +0.00000000000017,
+            +0.00000000000005,
+            -0.00000000000003,
+            +0.00000000000001,
+        )
+        w0 = np.real(sc.lambertw(np.exp(x)))
+        # For "very large values of x", sc.lambertw fails, instead the
+        # equation provided in the paper (see docstring) works better.
+        LIMIT_WHERE_TO_STOP_USING_LAMBERT_EQUATION_SOLVER = 400
+        large_x = x[x>LIMIT_WHERE_TO_STOP_USING_LAMBERT_EQUATION_SOLVER]
+        w0[x>LIMIT_WHERE_TO_STOP_USING_LAMBERT_EQUATION_SOLVER] = large_x-large_x*np.log(large_x)/(large_x+1)
+        
+        w4 = 2.9262710624435
+        Phi = np.polynomial.chebyshev.Chebyshev(dn)(2*w4/w0-1)
+        Phi[(x < 4)] = float('NaN')
+        return Phi
+
+    x_is_iterable = hasattr(x, '__iter__')
+    if not x_is_iterable:
+        x = [x]
+    x = np.asarray(x)
+
+    with np.errstate(over='ignore', invalid='ignore', divide='ignore'):
+        Phi = np.nansum(np.stack([f(x) for f in (Phi_0, Phi_1, Phi_2, Phi_3, Phi_4)]), axis=0)
+
+    if x_is_iterable:
+        return Phi
+    else:
+        return Phi[0]
+
+
+class landau_gen(rv_continuous):
+    r"""A Landau continuous random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `landau` (_[1], _[2]) is:
+
+    .. math::
+
+        f(x) = \frac{1}{2\pi i}\int_{a-i\infty}^{a+i\infty}e^{s\log(s)+xs}\ ds
+
+    for a real number :math:`x`.
+
+    %(after_notes)s
+
+    References
+    ----------
+    .. [1] Landau, L. (1944). "On the energy loss of fast particles by
+           ionization". J. Phys. (USSR). 8: 201.
+    .. [2] "Landau Distribution", Wikipedia,
+           https://en.wikipedia.org/wiki/Landau_distribution
+
+    %(example)s
+
+    """
+
+    def _shape_info(self):
+        return []
+
+    def _pdf(self, x):
+        s = (np.pi/2)**-1
+        x0 = 0.14182804795804668-0.42931453806624986
+        return _landau_pdf((x-x0)/s)/s
+    
+    def _cdf(self, x):
+        s = (np.pi/2)**-1
+        x0 = 0.14182804795804668-0.42931453806624986
+        return _landau_cdf((x-x0)/s)
+    
+    def _stats(self):
+        return np.nan, np.nan, np.nan, np.nan
+    
+    def _munp(self, n):
+        return np.nan
+    
+    def _fitstart(self, data, args=None):
+        # Initialize ML guesses using quartiles instead of moments.
+        if isinstance(data, CensoredData):
+            data = data._uncensor()
+        p25, p50, p75 = np.percentile(data, [25, 50, 75])
+        return p50, (p75 - p25)/2
+    
+    def _rvs(self, size=None, random_state=None):
```

Comment:
Turns out there's an efficient and relatively simple way to [generate random variates from stable distributions](https://www.jstor.org/stable/pdf/2285309.pdf?casa_token=XMi92GzUp9sAAAAA:VwyR-RlzSUZbnXYlFHJ_H6LlIvxJe_Vk-CSP7ba9uFVkY9zq7ylkRWBuATIhMC7sAhgv9JasHo-FV2QH6sSbR-ELOZdtWTXpdqFzAg7a8OygbUXbijkx). 

For Landau ($\alpha=\beta=1$):
<img width="487" alt="image" src="https://github.com/user-attachments/assets/9f81648c-bb87-47e0-9059-a3053ffd0018">
($W$ is standard exponential.)

```python3
from scipy.special import _ufuncs
import matplotlib.pyplot as plt

import numpy as np
rng = np.random.default_rng(2348923492)
size = 100000
pi_2 = np.pi/2

U = rng.uniform(-np.pi/2, np.pi/2, size=size)
W = rng.standard_exponential(size=size)
S = 2/np.pi * ((pi_2 + U) * np.tan(U) - np.log((pi_2*W*np.cos(U))/(pi_2 + U)))

x = np.linspace(-3, 20, 300)
pdf = _ufuncs._landau_pdf(x, 0, 1)
plt.plot(x, pdf)
plt.hist(S, density=True, bins=x[::5])
plt.show()
```

<img width="785" alt="image" src="https://github.com/user-attachments/assets/f064bedf-52bf-49af-8830-43cb2dc0f44d">